### PR TITLE
Small typo fix + two lipid identifiers

### PR DESCRIPTION
**Pull request by egonw**
Fixed a typo and adds fetch lipid identifiers from Wikidata.

PR task list:
- [x] Update NEWS
- [x] Add tests (if appropriate)
- [x] Update documentation with `devtools::document()`
- [x] Check package passed

### CODE REVIEW

**Comment by egonw** [comment]
I cannot run `R CMD check` as it does not finish in time. Please let me know how to use/do `devtools::document()`.

File: tests/testthat/test-wikidata.R [hunk 1]
```diff
@@ -42,6 +42,20 @@ test_that("wd_ident returns correct results", {
                            'drugbank', 'zvg', 'chebi', 'chembl', 'unii', 'source_url', 'query'))
```
Comment (Aariq) [inline tests/testthat/test-wikidata.R:42]: Can you also edit this test to include the new columns?  You might also consider combining your new test into this one in some way so there's less redundant testing.

```suggestion
                           'drugbank', 'zvg', 'chebi', 'chembl', 'unii', 'lipidmaps', 'swisslipids', 'source_url', 'query'))
```

**Comment by Aariq** [comment]
> I cannot run `R CMD check` as it does not finish in time. Please let me know how to use/do `devtools::document()`.

If you have the `devtools` package installed, just enter `devtools::document()` in the console.  Alternatively, in R Studio, you can use Document in the Build menu.

File: tests/testthat/test-wikidata.R (hunk 1 above)
Comment (egonw) [reply tests/testthat/test-wikidata.R:42]: I will combine them.

**Comment by Aariq** [comment]
It looks like a couple of the wikidata tests are still failing. Are they working locally on your machine?

https://github.com/ropensci/webchem/pull/310/checks?check_run_id=1389101163#step:9:328
https://github.com/ropensci/webchem/pull/310/checks?check_run_id=1389101163#step:9:339

**Comment by egonw** [comment]
Well, it did when I installed the packages. It seems that wil the other fails I overlooked this issue. I'll fix it. Sorry for the inconvenience.

**Comment by Aariq** [comment]
> Well, it did when I installed the packages. It seems that wil the other fails I overlooked this issue. I'll fix it. Sorry for the inconvenience.

No inconvenience at all!  If anything, this is our fault for having too much redundancy in our tests!  We're working on getting the tests to run faster, and making it easier for people to contribute, so this is all very helpful!

Please let us know if you have more questions or need help fixing the tests.  I can make more specific suggestions, but don't have time to take a look at it this week.  Thanks for putting in the work to improve webchem!

**Comment by stitam** [comment]
I am not sure why all these commits were added when I pushed my edits, I just wanted to rebase the PR on top of the current master.

**Pull request merged**